packages/credential-w3c/src/action-handler.ts

```diff
@@ -215,13 +215,52 @@ export interface IVerifyPresentationArgs {
    */
   fetchRemoteContexts?: boolean
 
+
+  /**
+   * Verification Policies for the verifiable presentation
+   * These will also be forwarded to the lower level module
+   */
+  policies?: VerifyPresentationPolicies
+
   /**
    * Other options can be specified for verification.
    * They will be forwarded to the lower level modules. that performt the checks
-   */
+    */
   [x: string]: any
 }
 
+export interface VerifyPresentationPolicies {
+  now?: number
+  nbf?: boolean
+  iat?: boolean
+  exp?: boolean
+}
+
+/**
+ * Encapsulates the response object to verifyPresentation method after verifying a
+ * {@link https://www.w3.org/TR/vc-data-model/#presentations | W3C Verifiable Presentation}
+ *
+ * @public
+ */
+ export interface IVerifyPresentationResponse {
```

Comment:
Let's make this object extensible to allow various other details to be included in the result:
```suggestion
 export interface IVerifyPresentationResponse {
   [x:string]: any
   
```

Reply:
Updated this as well.